packages/amplication-server/src/core/fsStorage/fsStorage.module.ts

```diff
@@ -0,0 +1,19 @@
+import { Module } from '@nestjs/common';
+import { FsStorageService, FS_STORAGE_SERVICE } from './fsStorage.service';
+
+@Module({
+  controllers: [],
+  providers: [
```

Comment:
We don't consume the service in this module, so it does not need to be included in `provider: []`

Reply:
Done